src/store/actions/types/sidebar.js

```diff
@@ -0,0 +1,4 @@
+export const SIDEBAR_LEFT_OPEN = 'SIDEBAR_KEFT_OPEN';
```

Comment:
it is possible to use redux ducks modules convention )

Reply:
good point

Reply:
But I can create separate issue for that point. I think it was better solution

Reply:
Yes, of course! no need to introduce it right now )